src/main/java/Commands/Command.java

```diff
@@ -0,0 +1,36 @@
+package Commands;
+
+import Duke.DukeException;
+import Duke.TaskList;
+import Duke.Ui;
+import Duke.Storage;
+
+/**
+ * Parent class of all Duke Commands
+ */
+public abstract class Command {
+    private boolean isExit;
+
+    public Command() {
+        this.isExit = false;
+    }
+
+    public void setExit() {
```

Comment:
What is the rationale for allowing `12: private boolean isExit` to be set by `12: public void setExit()`

Another alternative would be to default `12: public void setExit()` to false,
```java
// abstract class Command
public boolean isExit() {
    return false;
}
```

and override it in an exit command.

```java
// class ExitCommand
@Override
public boolean isExit() {
    return true;
}
```